index.html

```diff
@@ -48,25 +47,13 @@ <h2 class="subtitle">
         </div>
     </div>
 
-    <div class="box hero is-info has-text-centered-mobile is-vcentered">
-        <h1 class="title">
-            Webxdc can be anything
-        </h1>
-        <p>
-            Collaborative Tools as <br class="is-hidden-tablet">Editors, Todo-Lists<br>
-            Shared Calendars, Doodles<br>
-            Polls<br>
-            Games, of course
-        </p>
-    </div>
-
     <div class="box hero is-success">
         <div>
             <h1 class="title">
-                Try it out!
+                try it out, it's fun :) 
             </h1>
             <p>
-                Download <code>.xdc</code> from below and attach it to any chat in <a href="https://get.delta.chat">Delta Chat 1.30 or newer</a>.
+                install and configure <a href="https://delta.chat">delta chat</a>, download <code>.xdc</code> from below, attach it as a file to a chat and hit "start".
```

Comment:
```suggestion
                after installing <a href="https://delta.chat">delta chat</a>, download an <code>.xdc</code> file from below and send it in any chat, then hit "start".
```

Reply:
'install and configure' is accurate but sounds laborious, perhaps could be less technical overall…

Reply:
the problem is that "install and configure" is quite a barrier however you phrase it.  It will block out people who might otherwise be interested to try things out. I am wondering if we can publish a QR code for obtaining a "tmp.testrun.org" account where you can only message people on the "tmp" island for now. All outbound traffic could be blocked.